roles/scenario4/tasks/fix_controller.yml

```diff
@@ -1,4 +1,23 @@
 ---
+- name: Find neutron container name
```

Comment:
the name of this block is incorrect. Its goal is to set util_cmd, not to find a container name (if anything it looks for pod name, not container name)